hil-test/src/bin/interrupt.rs

```diff
@@ -95,29 +110,31 @@ mod tests {
 
     #[test]
     #[rustfmt::skip]
-    fn interrupt_latency(_ctx: Context) {
-        // unsafe {
-        //     asm!(
-        //         "
-        // csrrwi x0, 0x7e0, 1 #what to count, for cycles write 1 for instructions write 2
-        // csrrwi x0, 0x7e1, 0 #disable counter
-        // csrrwi x0, 0x7e2, 0 #reset counter
-        // "
-        //     );
-        // }
-
-        // // interrupt is raised from assembly for max timer granularity.
-        // unsafe {
-        //     asm!(
-        //         "
-        // li {bit}, 1                   # Flip flag (bit 0)
-        // csrrwi x0, 0x7e1, 1           # enable timer
-        // sw {bit}, 0({addr})           # trigger FROM_CPU_INTR0
-        // ",
-        //     options(nostack),
-        //     addr = in(reg) ctx.sw0_trigger_addr,
-        //     bit = out(reg) _,
-        //     )
-        // }
+    fn interrupt_latency(ctx: Context) {
+        unsafe {
+            asm!(
+                "
+                    csrrwi x0, 0x7e0, 1 #what to count, for cycles write 1 for instructions write 2
+                    csrrwi x0, 0x7e1, 0 #disable counter
+                    csrrwi x0, 0x7e2, 0 #reset counter
+                "
+            );
+        }
+        
+        // interrupt is raised from assembly for max timer granularity.
+        unsafe {
+            asm!(
+                "
+                    li {bit}, 1                   # Flip flag (bit 0)
+                    csrrwi x0, 0x7e1, 1           # enable timer
+                    sw {bit}, 0({addr})           # trigger FROM_CPU_INTR0
+                ",
+                options(nostack),
+                addr = in(reg) ctx.sw0_trigger_addr,
+                bit = out(reg) _,
+            )
+        }
+
+        defmt::info!("Performance counter: {}", unsafe { LAST_PERF });
```

Comment:
If this test doesn't actually assert anything, it should be in qa-test

Reply:
Please also explain what the heck is going on here. I look at this test, and see that you're reading a variable that the test doesn't set, and I don't want to reverse engineer magic.